rust/hook-janitor/src/webhooks.rs

```diff
@@ -394,7 +399,13 @@ impl WebhookCleaner {
 
         let (completed_row_count, completed_agg_row_count) = {
             let completed_row_count = self.get_row_count_for_status(&mut tx, "completed").await?;
-            let completed_agg_rows = self.get_completed_agg_rows(&mut tx).await?;
+            let completed_agg_rows = if self.hog_mode {
+                // Hog mode doesn't need to send metrics to Kafka (and can't aggregate by
+                // plugin anyway), so we can skip this.
+                vec![]
```

Comment:
When `app_metrics2` exists, we'll push some metrics there (in a followup PR). Even if they aren't useful to Hog or visible to customers they are useful for debugging rusty-hook.

Reply:
Makes sense 👍